src/internal/generated/styles/tokens.d.ts

```diff
@@ -61,3 +61,9 @@ export const colorChartsThresholdNegative: string;
 export const colorChartsThresholdPositive: string;
 export const colorChartsThresholdInfo: string;
 export const colorChartsThresholdNeutral: string;
+
+// Spacing
+export const spaceXs: string;
```

Comment:
We maintain a manually crafted `*.d.ts` declaration for our tokens, because the real token generation runs later in our build tasks. To break this circle, this file exists to make typescript build pass without dependency on style-dictionary -> theming build